dataservice/api/common/pagination.py

```diff
@@ -101,6 +102,17 @@ def indexd_pagination(q, after, limit):
 
     :returns: A Pagination object
     """
+    def prefetch_indexd():
+        """ Compute dids for the page and have indexd fetch them in bulk """
+        model = q._entities[0].mapper.entity
+        gfs = (q.order_by(model.created_at.asc())
+                .filter(model.created_at > after)
+                .with_entities(model.latest_did)
+                .limit(limit).all())
+        dids = [gf[0] for gf in gfs]
+        indexd.prefetch(dids)
+
+    prefetch_indexd()
```

Comment:
This looks like it always prefetches twice without needing to. I think you only need the one down inside of `while (pager.total > 0 and refresh)`